debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/BinlogReader.java

```diff
@@ -711,6 +712,7 @@ public void onCommunicationFailure(BinaryLogClient client, Exception ex) {
 
         @Override
         public void onEventDeserializationFailure(BinaryLogClient client, Exception ex) {
+            logger.debug("A deserialization failure event arrived", ex);
```

Comment:
Is the MDC context already set on this thread; e.g. are these methods called from the binlog thread? If not, we probably need more context if these are to be useful when this connector is deployed alongside other connectors.

Reply:
It is set by `onConnect` method
An example of log message
```
2017-09-04 05:13:10,362 DEBUG  MySQL|dbserver1|binlog  A communication failure event arrived   [io.debezium.connector.mysql.BinlogReader]
java.io.EOFException
        at com.github.shyiko.mysql.binlog.io.ByteArrayInputStream.read(ByteArrayInputStream.java:190)
        at com.github.shyiko.mysql.binlog.io.ByteArrayInputStream.readInteger(ByteArrayInputStream.java:46)
        at com.github.shyiko.mysql.binlog.event.deserialization.EventHeaderV4Deserializer.deserialize(EventHeaderV4Deserializer.java:35)
        at com.github.shyiko.mysql.binlog.event.deserialization.EventHeaderV4Deserializer.deserialize(EventHeaderV4Deserializer.java:27)
        at com.github.shyiko.mysql.binlog.event.deserialization.EventDeserializer.nextEvent(EventDeserializer.java:175)
        at io.debezium.connector.mysql.BinlogReader$1.nextEvent(BinlogReader.java:116)
        at com.github.shyiko.mysql.binlog.BinaryLogClient.listenForEventPackets(BinaryLogClient.java:741)
```